SETUP.md

```diff
@@ -0,0 +1,32 @@
+# Installasjonsguide for workshop
+Guide for installasjon av editor, pakkebehandler og git.
+
+## Editor:
+Når det gjelder Editor står dere fritt til å velge selv, men f.eks VS Code er mye brukt blant frontendutviklere og lar deg installere støtte for Svelte.
+
+Følg instruksjonene her for å installere Visual Studio Code: https://code.visualstudio.com/download
+
+## Pakkebehandler
+Du må ha en pakkebehandler installert på maskinen for å bygge og kjøre prosjektet lokalt. 
+
+### Mac
+Vi anbefaler å bruke HomeBrew som er en pakkebehandler for Mac: https://brew.sh/index_nb. Du kan installere denne på din maskin ved å åpne en terminal og kopiere inn følge kommando: 
+
+``` /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" ```
```

Comment:
Dette parses feil fordi du bruker tre backticks i stedet for én. Tre backticks indikerer en flerlinjers kodeblokk, der man eventuelt angir språket på samme linje som de første tre backticks.